test_scripts/test_setups.py

```diff
@@ -195,7 +199,10 @@ class TestPhase(Enum):
             }
 
 _telluric = {'gemini_gnirs/32_SB_SXD':
-                 {'coadd_file': 'pisco_coadd.fits', 'redshift': 7.52, 'objmodel': 'qso'},
+#                 {'coadd_file': 'pisco_coadd.fits', 'redshift': 7.52, 'objmodel': 'qso'},
```

Comment:
Remove the commented out line if you're sure there's no need to go back to doing it the other way

Reply:
Done!  Thanks for the review!